modules/fieldlinessequence/rendering/renderablefieldlinessequence.h

```diff
@@ -192,29 +209,25 @@ class RenderableFieldlinesSequence : public Renderable {
     // Index of the extra quantity to use for masking
     properties::OptionProperty _pMaskingQuantity;
 
-// Button which sets camera focus to parent node of the renderable
-    properties::TriggerProperty _pFocusOnOriginBtn;
+    /// Line width for the line rendering part
+    properties::FloatProperty _pLineWidth;
+
     // Button which executes a time jump to start of sequence
     properties::TriggerProperty _pJumpToStartBtn;
 
     // --------------------- FUNCTIONS USED DURING INITIALIZATION --------------------- //
     void addStateToSequence(FieldlinesState& STATE);
     void computeSequenceEndTime();
     void definePropertyCallbackFunctions();
-    bool extractCdfInfoFromDictionary(std::string& seedFilePath, std::string& tracingVar,
-        std::vector<std::string>& extraVars);
-    bool extractJsonInfoFromDictionary(fls::Model& model);
-    void extractMagnitudeVarsFromStrings(std::vector<std::string>& extraVars,
-        std::vector<std::string>& extraMagVars);
-    bool extractMandatoryInfoFromDictionary(SourceFileType& sourceFileType);
-    void extractOptionalInfoFromDictionary(std::string& outputFolderPath);
-    void extractOsflsInfoFromDictionary();
-    bool extractSeedPointsFromFile(const std::string& path,
-        std::vector<glm::vec3>& outVec);
+    //bool extractCdfInfoFromDictionary();
+    fls::Model extractJsonInfoFromDictionary();
+    void extractMagnitudeVarsFromStrings(std::vector<std::string>& extraMagVars);
```

Comment:
Consider returning the `std::vector<std::string>` here instead as it is easier to understand